src/python/pypresso.cmakein

```diff
@@ -15,15 +16,53 @@ else
 fi
 export PYTHONPATH
 
-if [ "$1" = "--gdb" ]; then
-  shift
-  exec gdb -ex run --args @PYTHON_FRONTEND@ $@
-elif [ "$1" = "--lldb" ]; then
-  shift
-  exec lldb -o "run $@" @PYTHON_FRONTEND@
-elif [ "$1" = "--cuda-gdb" ]; then
-  shift
-  exec cuda-gdb -ex run --args @PYTHON_FRONTEND@ $@
-fi
-
-exec @PYTHON_FRONTEND@ $@
+case $1 in
+    --gdb)
+        shift
+        exec gdb --args @PYTHON_FRONTEND@ $@
+        ;;
+    --lldb)
+        shift
+        exec lldb -- @PYTHON_FRONTEND@ $@
```

Comment:
Have you tested whether this works? Previously we supplied the arguments to the `run` command.

 Is there any reason why you no longer have the debuggers run the program directly? You can‘t even set breakpoints before the first run because we use dynamic libraries, so you might as well run immediately.

Reply:
This form of the `lldb` command is what I found [online](https://lldb.llvm.org/lldb-gdb.html).

No, you can set breakpoints without shared libraries loaded (only tab-completion won't work).
```
(gdb) break integrate_vv
Function "integrate_vv" not defined.
Make breakpoint pending on future shared library load? (y or [n]) y
Breakpoint 1 (integrate_vv) pending.
```